### DISCUSSION TITLE
How to invite people in my private repository?

### DISCUSSION THREAD

**Discussion by crypto-demigod** [community/community#33402] [Other Feature Feedback, Questions, & Ideas] [answered]
Dear folks.
How to invite people in my private repository?
I have a project but I need some people then what shall I do?

**Comment by massimo1220** [comment, marked as answer]
Hi, @superherodemigod , glad to hear from you,
The answer is simple.
Ask for the username of the person you're inviting as a collaborator. ...
On GitHub.com, navigate to the main page of the repository.
Under your repository name, click Settings.
In the "Access" section of the sidebar, click Collaborators & teams.
Click Invite a collaborator.

**Reply by crypto-demigod** [reply]
thanks.@devstar0202.

**Reply by massimo1220** [reply]
my pleasure.

**Reply by TarikSaran** [reply]
I tried this and it sends an email to my invitee and when he clicks on accept invite it takes him to a page with error 404. Is there another way to accept invites other then thru email?

**Reply by massimo1220** [reply]
It's better if your friend checks the notification in the top right corner...

**Comment by anhtran917** [comment]
Hi, there.
To invite people to your private repository and collaborate on your project, you can follow these steps:

- Access your repository on the Github.
- Navigate to the settings of your repository.
- Look for the "Collaborators" tab in right menu and click "Add people".
- Enter the usernames or email addresses of the people you want to invite as collaborators.
- Specify the appropriate access level for each collaborator (e.g., read-only access, read-write access).
- Send the invitation or save the changes to invite the selected individuals.
The invited people will receive notifications or emails with the invitation to join your private repository. Once they accept the invitation, they will be able to access and contribute to your project.

I hope this answer will be helpful for you.
Thank you.
